src/pages/Check.jsx

```
@@ -6,15 +6,23 @@ import { Button, Input, Modal, Text } from '../components'
 import ticket from '../assets/ticket.png'
 import warning from '../assets/warning.png'
 import arrow from '../assets/arrow-left.svg'
-
+import { useGetPackageInformation } from '../hooks'
 const Check = () => {
   const [packageCode, setPackageCode] = useState('')
   const [isModalOpen, changeIsModalOpen] = useState(true)
   const history = useHistory()
+  const { packageInformation } = useGetPackageInformation({ packageId: packageCode })
 
-  const toDelivery = e => {
+  const toDelivery = async e => {
     e.preventDefault()
-    history.push('/delivery/test')
+    const { packages } = await packageInformation
+    if ((await packages.length) > 0) {
+      history.push(`/delivery/${packageCode}`)
+    }
+
+    if ((await packages.length) === 0) {
+      alert('el packcode no existe')
```

use log Error

---

but this is not an error, it should throw an error message that the id does not exist or something

---

Got it, maybe in the future we can implement a toast for this

For the moment being, is OK like this